pythonforandroid/recipes/hostpython3/__init__.py

```diff
@@ -67,7 +67,7 @@ def get_recipe_env(self, arch=None):
         openssl_prereq = OpenSSLPrerequisite()
         if env.get("PKG_CONFIG_PATH", ""):
             env["PKG_CONFIG_PATH"] = os.pathsep.join(
-                openssl_prereq.pkg_config_location, env["PKG_CONFIG_PATH"]
+                openssl_prereq.pkg_config_location  # , env["PKG_CONFIG_PATH"]
             )
```

Comment:
Ouch. I guess I did a typo, but `env["PKG_CONFIG_PATH"]` should stay here.
Can you please change it to something like ?
```python
env["PKG_CONFIG_PATH"] = os.pathsep.join(
    [openssl_prereq.pkg_config_location, env["PKG_CONFIG_PATH"]]
)
```

Reply:
who has never used this will have a mistake